docs/api.asciidoc

```diff
@@ -26,14 +26,15 @@ entirely.
 
 [float]
 == APIs
-
+* <<spaces-api>>
 * <<role-management-api>>
 * <<saved-objects-api>>
 * <<dashboard-import-api>>
 * <<logstash-configuration-management-api>>
 * <<url-shortening-api>>
 --
 
+include::api/spaces-management.asciidoc[]
 include::api/role-management.asciidoc[]
 include::api/saved-objects.asciidoc[]
 include::api/dashboard-import.asciidoc[]
```

Comment:
For the APIs, we have an intro page, and then put each API on its own page.  So Update space, Create Space, Get Space, and Delete Space would each have its own page.  You can see an example [here](https://www.elastic.co/guide/en/kibana/current/role-management-api.html).

Reply:
I'm having a hard time figuring out what went wrong. In my published version (https://legrego.github.io/kibana-spaces-docs/index.html), both the role management api and the spaces management api are missing their intro page. I think I created the space management api docs in the same format as the role management api docs, so I'm wondering if it's just an issue with the way I'm generating the HTML docs for the preview?

Reply:
@legrego I built the docs and everything looks good.  Intro pages are there.  So it must just be a problem with the preview.